app/views/widgets/onlineTestProgress.scala.html

```diff
@@ -190,19 +191,25 @@
                                             Continue section</button>
                                     }
                                 }
-                                <p>(You'll have 80 minutes to finish)</p>
+                                <p>(You'll have @{dashboardPage.phase2TimeToCompleteTest.getOrElse(80) } minutes to finish)</p>
                             </div>
                         </div>
                     </li>
                 }
+                <li id="videoSection">
```

Comment:
This is a duplicate of line 225 onward. I think it may have been resolved in master though, so perhaps merging the changes will fix this